Sources/XMTP/Codecs/RemoteAttachmentCodec.swift

```diff
@@ -206,4 +206,8 @@ public struct RemoteAttachmentCodec: ContentCodec {
 
 		return Data(parameterData)
 	}
+	
+	public func shouldPush(content: RemoteAttachment) throws -> Bool {
+		return false
```

Comment:
Do we not want to push for remote attachments?

Reply:
the scope of the task only includes `textCodec` and `Composite`

Reply:
👋 @kele-leanes I actually shared a table with you that showed the rest of the codecs as well. We definitely want this to be true. Just because JS has codecs in a different repo doesnt mean they aren't also supported. Here is the table for reference again.

<img width="385" alt="Screenshot 2024-01-30 at 7 26 54 AM" src="https://github.com/xmtp/xmtp-ios/assets/3522670/a4183065-dca1-4a7f-ad37-07262291ec61">